frontend/src/app/header/header.component.html

```diff
@@ -2,7 +2,7 @@
   <span class="navbar-brand">ApiCenter</span>
 
   <ul class="navbar-nav mr-auto" *ngIf="username">
-    <li class="nav-item"><a routerLink="/">Home</a></li>
+    <li class="nav-item"><a href="/">Home</a></li>
```

Comment:
Why this change?

Reply:
Fixed now 🎉 So the change is removed. (GraphiQL messed up the router history, so neither this Home link nor the browser back button worked. Since figured out why. This was a workaround, so the link still worked.)